OWNERS_ALIASES

```diff
@@ -14,14 +14,9 @@ aliases:
     - anjannath
     - gbraad
     - evidolob
-  doc-reviewers:
-    - cfergeau
-    - praveenkumar
-    - anjannath
-    - gbraad
-    - evidolob
-    - adrianriobo
-    - themr0c
   test-reviewers:
     - adrianriobo
+    - lilyLuLiu
+    - albfan
+    - lstocchi
```

Comment:
`lstocchi` is an engineer, not a test-reviewer. otherwise, we should put everyone in that group.